torch/testing/__init__.py

```diff
@@ -5,6 +5,8 @@
 import torch
 import random
 
+from .FileCheckBuilder import FileCheckBuilder as FileCheckBuilder
```

Comment:
Maybe should live in `torch.jit.testing` instead since it won't be used outside of jit testing much